jenkins-builds/6.0.UP04-17.08-gpu

```diff
@@ -34,5 +34,6 @@
  Score-P-3.1-CrayPGI-17.08.eb
  Spark-1.6.0.eb                                     --set-default-module
  vampir-9.3.0.eb                                    --set-default-module
+ VASP-5.4.4-CrayIntel-17.08-cuda-8.0.eb --set-default-module
```

Comment:
align please

Reply:
To what? I see several different alignments in the file

Reply:
the rightmost please

Reply:
Tabs vs. spaces problem 😂